package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "build": "npm run style:remove && npm run style:compile && npm run style:rename && npm run style:postCSS && npm run style:assets",
-    "build:prod": "npm run build && npm run style:minify && npm run style:less",
+    "build:prod": "npm run build && npm run style:less && npm run style:minify",
```

Comment:
The less task needs to run before minify so that `fundamentals.less.min.css` and `fundamentals-ie11.less.min.css` aren't created in the `dist` folder.